client/src/utils/PostCallUtil.ts

```diff
@@ -22,6 +22,7 @@ export const submitSurveyResponseUtil = async (
       })
     });
     if (res.status !== 200) {
+      console.log('Error during insertion');
```

Comment:
Is this logging for testing and should be removed?

Reply:
I added this logging function just so Contoso would get some indication that something went wrong. My thought was to have a minimized version of Error Handling of sorts. Do you think this would be bad practice? I can remove it if so.

Reply:
I don't think it is bad to log to console. Only that it might get noisy for the developer. Ideally we would have a logging system where developers can configure the level of logging and have a way to reduce the noise. But we don't have that right now and also it might look like we forgot to remove a debug console.log.

Reply:
ok, removed the log and just added a comment instead